src/language-server/project/base.ts

```diff
@@ -98,9 +97,11 @@ export abstract class GraphQLProject implements GraphQLSchemaProvider {
     // if a config doesn't have a uri associated, we can assume the `rootURI` is the project's root.
     this.rootURI = config.configDirURI || configFolderURI;
 
-    const { includes, excludes } = config.isClient
+    const { includes = [], excludes = [] } = isClientConfig(config)
       ? config.client
-      : config.service;
+      : {
+          /** TODO */
```

Comment:
Should this TODO be captured in an issue? You probably have it tracked somewhere, asking just in case :)